icon-two-consumer-models/pom.xml

```diff
@@ -10,7 +10,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>icon-two-consumer-models</artifactId>
-
+    <version>1.0.0</version>
```

Comment:
is this needed?

Reply:
yes, it needs to stop connecting RabbitMQ in unit test